### PR TITLE
add UserSimpleView add Qonsoll Grid and Theme add antIcons

### PR DESCRIPTION
**Pull request by okrdima**
image
add UserSimpleView add Qonsoll Grid and Theme add antIcons

### CODE REVIEW

File: src/app/index.js [hunk 1]
```diff
@@ -1,5 +1,14 @@
+import { ThemeProvider } from '@qonsoll/react-design'
+import Theme from '../config/theme'
+import '../config/root.scss'
+import 'antd/dist/antd.css'
+
 function App() {
-  return <div>Hello</div>
```
Comment (Slavka406) [inline src/app/index.js:2]: Use Box, please

File: src/app/index.js [hunk 2]
```diff
@@ -1,5 +1,14 @@
...
+  return (
+    <ThemeProvider theme={Theme}>
+      <div>Hello app</div>
```
Comment (Slavka406) [inline src/app/index.js]: same

File: src/config/root.scss [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+  --border-radius-xxl: 64px;
+
+  // z-indexes
+  --mobile-stepper-z-index: 1000;
+  --app-bar-z-index: 1100;
+  --drawer-z-index: 1200;
+  --modal-z-index: 1300;
+  --snackbar-z-index: 1400;
+  --tooltip-z-index: 1500;
+}
```
Comment (Slavka406) [inline src/config/root.scss]: Add newline at end of file, please

File: src/app/domains/user/components/views/UserSimpleView/UserSimpleView.template.js [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+  const { avatarURL, withName, name, avatarSize, withEmail, email } = props
+
+  return (
+    <Row v="center" noOuterGutters>
+      <Col cw="auto">
+        {/*icon if user without avatar*/}
+        <Avatar size={avatarSize} src={avatarURL} icon={<UserOutlined />} />
+      </Col>
+      <Col cw="auto" px={0}>
+        <Row>{withName && <Text>{name}</Text>}</Row>
```
Comment (Slavka406) [inline src/app/domains/user/components/views/UserSimpleView/UserSimpleView.template.js]: Fix grid, please

Comment (sashka2131) [inline src/app/domains/user/components/views/UserSimpleView/UserSimpleView.template.js:17]: U should use Row with Col

**Review by Slavka406 — APPROVED** [review]

**Pull request merged**